examples/Server/Server.csproj

```diff
@@ -25,4 +25,8 @@
     <PackageReference Include="Grpc.Tools" Version="$(GrpcToolsPackageVersion)" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Properties\" />
```

Comment:
remove?

Reply:
Ugh, I hate how VS automatically adds launchsettings when you debug with F5. The file is ignored by git but VS still creates a folder in the project.